apisix/plugins/ai-proxy-multi.lua

```diff
@@ -215,7 +215,7 @@ local function proxy_request_to_llm(conf, request_table, ctx)
     local ai_driver = require("apisix.plugins.ai-drivers." .. provider)
     local res, err, httpc = ai_driver:request(conf, request_table, extra_opts)
     if not res then
-        if (ctx.balancer_try_count or 0) < 1 then
+        if (ctx.ai_balancer_try_count or 0) < 2 then
```

Comment:
@membphis Do we need to make the number of retries a plugin configuration?

Reply:
we can implement this feature later